examples/connect.rs

```diff
@@ -115,24 +119,21 @@ mod codec {
 }
 
 mod tcp {
-    use std::io;
-    use std::net::SocketAddr;
+    use tokio;
+    use tokio::net::TcpStream;
+    use tokio::prelude::*;
 
     use bytes::BytesMut;
-    use futures::{Future, Stream};
-    use futures::future::Executor;
-    use futures_cpupool::CpuPool;
-    use tokio::net::TcpStream;
-    use tokio_io::AsyncRead;
     use codec::Bytes;
 
+    use std::io;
+    use std::net::SocketAddr;
+
     pub fn connect(addr: &SocketAddr,
-                   pool: &CpuPool,
                    stdin: Box<Stream<Item = Vec<u8>, Error = io::Error> + Send>)
-        -> Box<Stream<Item = BytesMut, Error = io::Error>>
+        -> Box<Stream<Item = BytesMut, Error = io::Error> + Send>
```

Comment:
IoStream?

Reply:
I don't like using the type aliases as they tend to lead to confusion. I rather be explicit here and at some point this will be replaced by `impl Trait`.